src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -60,15 +60,17 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Core" Version="1.9.0" />
```

Comment:
You shouldn't need a direct core dependency, storage would bring one for you. You shouldn't need Azure.Storage.Blobs, as well, it would be provided by Microsoft.Azure.WebJobs.Host.Storage